# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dvbd/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dvbd-0.7.6/src/outputbuffer.cpp.vanilla	2005-04-15 17:38:50.000000000 +0200
+++ dvbd-0.7.6/src/outputbuffer.cpp	2005-04-15 17:38:42.000000000 +0200
@@ -26,6 +26,8 @@
 #include <sys/socket.h>
 #include <sys/un.h>
 
+#include <errno.h>
+
 OutputBuffer::OutputBuffer(int fd, int initSize, int maxSize)
   : CircularBuffer(initSize, maxSize), fd(fd)
 {
